docs/src/main/paradox/configuration.md

```diff
@@ -49,3 +49,27 @@ Alternatively, Akka Discovery can be used for finding the Cassandra server conta
 in the @extref:[Alpakka Cassandra documentation](alpakka:cassandra.html#using-akka-discovery).
 
 Without any configuration it will use `localhost:9042` as default.
+
+## Using Cassandra-as-a-Service
+
+You can spin up a free Apache Cassandra cluster in the cloud using [DataStax
+Astra](https://www.datastax.com/products/datastax-astra). To use a Cassandra-as-a-Service cluster as Akka persistence,
+specify your secure connect bundle and credentials in your configuration instead of the contact points:
+
+```
+datastax-java-driver {
+  basic {
+    session-keyspace = my_keyspace
+    cloud {
+      secure-connect-bundle = /path/to/secure-connect-database_name.zip
+    }
+  }
+  advanced {
+    auth-provider {
+      class = PlainTextAuthProvider
+      username = user_name 
+      password = p@ssword1
```

Comment:
Looks like a fake password but maybe make it more clear with
```
username = <user name>
password = <password>
```

Reply:
Makes sense, I've changed it.